### DISCUSSION TITLE
Self-hosted installation...UI elements not showing?

### DISCUSSION THREAD

**Discussion by jill88** [OpnForm/OpnForm#621] [Q&A]
It seems to be that several UI elements are not showing on my installation.  Everything is working fine, but it seems like things like bullet toggles, and other UI elements, like tag box fields (Name, Email...etc for the email integration) just are not showing.  I tried in firefox and chrome and turned off ad blockers, and still the same.  And then one time they did show, but I don't know what is happening.  Usually, they are not showing.

**Comment by JhumanJ** [comment]
Fixed with latest version!
